contracts/0.8.25/vaults/Delegation.sol

```diff
@@ -340,6 +342,20 @@ contract Delegation is Dashboard {
         _voluntaryDisconnect();
     }
 
+    /**
+     * @notice Pauses deposits to beacon chain from the StakingVault.
+     */
+    function pauseBeaconChainDeposits() external onlyRole(CURATOR_ROLE) {
```

Comment:
Shouldn't it be in the Dashboard? And the getter too?

Reply:
Added to dashboard in https://github.com/lidofinance/core/pull/922/commits/765a205487760b39e551eb3db389329f05707b58